### DISCUSSION TITLE
¿Terminamos los casos de usos mientras el diseño de la pagina web aún esta en desarrollo?

### DISCUSSION THREAD

**Discussion by Curcolor** [Curcolor/COMMUNITY#10] [Q&A] [answered]
esta pregunta va más que nada a @SaryNotfound pues es la encargada del UX de la pagina, recuerda lo que paso con el proyecto OpticSoft, si se terminan los casos de uso primero tendrás que seguir el diagrama para hacer el diseño o esperamos que termines el diseño y vamos arreglando los casos de usos.

**Comment by SaryNotfound** [comment]
Yo diría que se termine primero el diseño y luego los casos de usos, porque después no concuerda el diseño  con los casos de uso y puede generar conflicto a la hora de programar o crear los otros diagramas.

**Reply by JUNIORRDSR** [reply]
pero ya yo llevo la mitad de los casos de uso :shipit:

**Reply by Curcolor** [reply]
pues entonces que lo terminamos y luego lo adaptamos?

**Reply by SaryNotfound** [reply, marked as answer]
no, dejen hasta ahí y luego adaptamos

**Reply by JUNIORRDSR** [reply]
@Curcolor entonces vamos hacer la arquitectura 🔪
